src/components/ActionCard.tsx

```diff
@@ -77,12 +76,8 @@ const ActionCard: React.FC<IProps> = ({
       >
         <Image
           src={image}
-          width={320}
+          width={220}
```

Comment:
This will cause problems in other places where we use the `ActionCard`, such as the home page.

We need to refactor the `ActionCard` to accept the image as a prop in order to pass the necessary image config from the consumer side.

```ts
// something like
<ActionCard image={<Image width={320 or 220 or whatever}>}>
```

Same problem occurs in the `PageHero` component.

I'd say to leave this as it is and tackle these refactors in a new PR.